### DISCUSSION TITLE
使用罗技G603鼠标手势延迟很严重。

### DISCUSSION THREAD

**Discussion by ContinueStudio** [shuax/MouseInc.Settings#214] [Q&A]
已经把灵敏度调成高了，使用手势时依然有很明显的延迟。希望能在灵敏度设置里增加一个特别高或者非常高的选项。感激不对劲~

**Comment by shuax** [comment]
4k显示器吗？卡估计是绘制轨迹太慢。

**Reply by ContinueStudio** [reply]
很感谢大佬的更新，目前这个问题已经解决了。我用了这个软件一年多了，反馈过几个问题，都得到了解决，真的很感谢你，也希望这个软件能走得更远。

下面有两个关于截图贴图的功能建议：
1.增加截图后编辑的功能，类似QQ和微信（简单的编辑即可，标注，箭头，写字）
2.贴图的快捷键希望能优化一下，默认鼠标滚轮是修改不透明度，感觉缩放的需求会更大一些。
   希望能改成默认缩放，按住Ctrl+滚轮是更改不透明度。

之前截图软件用的是Snipaste，因为它是专业做截图，可能功能上会更丰富一些，虽然Mouselnc的截图贴图只是一个其中一个小功能，但是能附加一些snipaste的精髓，也能让Mouselnc锦上添花。

因为不是程序员，所以不了解这两项功能添加的难易程度，只是从使用者的角度提出建议。
如果可以且不麻烦的话，希望能添加这两项升级。真的万分感谢。
